src/stores/messages/MessagesStore.ts

```diff
@@ -440,6 +445,7 @@ export default class MessagesStore {
 		const messageIndex = this.messagesIds.indexOf(messageId);
 		if (messageIndex !== -1) {
 			this.scrolledIndex = new Number(messageIndex);
+			this.highlightedMessageId = messageId;
```

Comment:
that's not a place to set highlighted message, instead we should do it in `onSavedItemSelect`

Reply:
Fixed